web3-utils/utils/deploy.ts

```diff
@@ -0,0 +1,48 @@
+import Web3 from 'web3'
+import contract from "../build/contracts/Migrations.json"
```

Comment:
We should not rely on `Migrations` only. And make this deployment script generic if possible.
As in the future we might want to deploy new contracts and we will need to make a different file or edit this one significantly.

Reply:
I made it more generic!